bin/react-to-product-owners-yml-changes.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+set -euo pipefail
```

Comment:
nit: `set -euox pipefail` will make the logs a bit nicer for us if things ever break.